core/capabilities/ccip/delegate.go

```diff
@@ -49,6 +51,15 @@ type RelayGetter interface {
 	GetIDToRelayerMap() (map[types.RelayID]loop.Relayer, error)
 }
 
+type Keystore[K keystore.Key] interface {
+	Get(id string) (K, error)
+	GetAll() ([]K, error)
+	Create(context.Context) (K, error)
+	Delete(ctx context.Context, id string) (K, error)
+	Import(ctx context.Context, keyJSON []byte, password string) (K, error)
+	Export(id string, password string) ([]byte, error)
+}
```

Comment:
Don't think you need all of these methods - `GetAll` is enough?

Reply:
can you use `web.Keystore` directly?

Reply:
I got a circular import error

Reply:
Oh too bad. In that case I would agree with @makramkd, you can narrow the interface to just the function we need